src/Symfony/Component/Process/Messenger/ProcessMessage.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Process\Messenger;
```

Comment:
I would move it under the Messenger namespace instead. I think it will make discoverability a bit easier.

Reply:
Ok, should I do the same for #49815 and #49814?